### PR TITLE
ADD new mongo driver in ci image build script

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: ci/rpm7/build-dep.sh [hunk 1]
```diff
@@ -44,24 +45,39 @@ yum -y install \
...
 && scons install --disable-warnings-as-errors --prefix=/usr/local --use-sasl-client --ssl \
 && rm -Rf /opt/mongo-cxx-driver-legacy-1.1.2
 
-curl -L https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz | tar xzC /opt/ \
+echo "INSTALL: mongodb c driver (required by mongo c++ driver)" \
+&& curl -L https://github.com/mongodb/mongo-c-driver/releases/download/1.16.0/mongo-c-driver-1.16.0.tar.gz | tar xzC /opt/ \
+&& cd /opt/mongo-c-driver-1.16.0 \
+&& mkdir cmake-build \
+&& cd cmake-build \
+&& cmake3 -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF .. \
```
Comment (fgalan) [inline ci/rpm7/build-dep.sh:61]: NO or OFF?

Check: http://mongoc.org/libmongoc/current/init-cleanup.html#deprecated-feature-automatic-initialization-and-cleanup

Comment (fgalan) [reply ci/rpm7/build-dep.sh:61]: OFF is fine. See https://jira.mongodb.org/browse/CDRIVER-3903

**Comment by mapedraza** [comment]
LGTM

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**